Packs/Core/Triggers/Trigger_-_Netcat_Makes_or_Gets_Connections.json

```diff
@@ -0,0 +1,19 @@
+{
+    "trigger_id": "907c5db410d816a487249e77cbbf411a",
+    "playbook_id": "Netcat Makes or Gets Connections",
+    "suggestion_reason": "Recommended for `Netcat Makes or Gets Connections` Alerts ",
+    "description": "This trigger is responsible for handling `Netcat Makes or Gets Connections` alert",
+    "trigger_name": "Netcat Makes or Gets Connections",
+    "fromVersion": "8.8.0",
+    "alerts_filter": {
+        "filter": {
+            "AND": [
+                {
+                    "SEARCH_FIELD": "alert_name",
+                    "SEARCH_TYPE": "CONTAINS",
```

Comment:
@efelmandar 
Please change to EQ and put the exact alert name (test it also to validate it works)